modules/nf-core/xengsort/index/main.nf

```diff
@@ -0,0 +1,54 @@
+process XENGSORT_INDEX {
+    tag "$host_fasta"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/xengsort:2.0.5--pyhdfd78af_0':
+        'biocontainers/xengsort:2.0.5--pyhdfd78af_0' }"
+
+    input:
+    path(host_fasta, stageAs: "host/*")
```

Comment:
I think allowing meta data here would be useful. It allows module users to process more than one sample if necessary.
```suggestion
    tuple val(meta), path(host_fasta, stageAs: "host/*")
```
And as such the `meta` field should be added in the appropriate places, e.g. in the output, noted in the meta.yml, etc.

Reply:
I would rather not include meta in this part of process, this is meant to be similar to `salmon/index`. In next part of process that I'm going to add in advance `xengsort/classify` I will include a meta file. What you think?

Reply:
I guess it depends on use case. As I'm not familiar with the tool or the field this is used in, I'm happy to leave this as is. If someone does need meta data passed through this, they can make a PR to add it later.